src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/ErrorReportHistory/__tests__/ErrorReportHistoryViewModelTests.swift

```diff
@@ -0,0 +1,44 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class ErrorReportHistoryViewModelTests: XCTestCase {
+
+	let model = ErrorReportHistoryViewModel()
```

Comment:
I'm not sure if a common model is helping here. We'll see cases where a custom model is created and then devs might get confused. I'd prefer one model per test case and keep the amount of properties low.

Reply:
Okay. I will move them to specific tests.